Packs/PrismaCloudCompute/Integrations/PaloAltoNetworks_PrismaCloudCompute/PaloAltoNetworks_PrismaCloudCompute.py

```diff
@@ -1572,6 +1625,101 @@ def get_impacted_resources(client: PrismaCloudComputeClient, args: dict) -> Comm
         raw_response=raw_response
     )
 
+def get_waas_policies(client: PrismaCloudComputeClient, args: dict) -> CommandResults:
+    """
+    Get the WAAS policies.
+    Implement the command 'prisma-cloud-compute-get-waas-policies'
+
+    Args:
+        client (PrismaCloudComputeClient): prisma-cloud-compute client.
+        args (dict): prisma-cloud-compute-get-waas-policies command arguments
+
+    Returns:
+        CommandResults: command-results object.
+    """
+    policies = client.get_waas_policies()
+
+    rules = policies["rules"][0]["applicationsSpec"][0]
```

Comment:
why do you specifically take only the first value of each object?

Reply:
Thanks, good catch! I have made this a loop now instead to handle multiple policies being already configured.